Kernel/Graphics/Definitions.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+
+namespace Kernel::Graphics {
+
+// Note: Address 0x50 is expected to be the DDC2 (EDID) i2c address.
+static constexpr u8 ddc2_i2c_address = 0x50;
```

Comment:
Can we squash `Kernel/Graphics: Export DDC I2C address definition to global header file` as well?

Kernel/Graphics/Definitions.h was deleted in ` Kernel/Graphics: Introduce the IntelDisplayConnectorGroup class ` and is resurrected here.